sources/packages/api/src/database/entities/student-assessment.model.ts

```diff
@@ -0,0 +1,208 @@
+import {
+  Column,
+  Entity,
+  JoinColumn,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+  RelationId,
+} from "typeorm";
+import {
+  Application,
+  AssessmentStatus,
+  EducationProgramOffering,
+  StudentAppeal,
+} from ".";
+import { ColumnNames, TableNames } from "../constant";
+import { AssessmentTriggerType } from "./assessment-trigger.type";
+import { RecordDataModel } from "./record.model";
+import { StudentScholasticStanding } from "./student-scholastic-standing.model";
+
+/**
+ * Represents all the assessments and reassessments processed for a particular
+ * Student Application. When a assessment/reassessment is needed it will use
+ * the information present in the application itself combined with the additional
+ * data present in this table (e.g. student appeals and scholastic standing changes).
+ */
+@Entity({ name: TableNames.StudentAssessment })
+export class StudentAssessment extends RecordDataModel {
+  @PrimaryGeneratedColumn()
+  id: number;
+  /**
+   * Application id related to this assessment.
+   */
+  @RelationId(
+    (studentAssessment: StudentAssessment) => studentAssessment.application,
+  )
+  applicationId: number;
+  /**
+   * Application related to this assessment.
+   */
+  @ManyToOne(() => Application, {
+    eager: false,
+    cascade: false,
+    nullable: false,
+  })
+  @JoinColumn({
+    name: "application_id",
+    referencedColumnName: ColumnNames.ID,
+  })
+  application: Application;
+  /**
+   * Workflow instance that processed this assessment.
+   */
+  @Column({
+    name: "assessment_workflow_id",
+    nullable: true,
+  })
+  assessmentWorkflowId?: number;
+  /**
+   * Represent the output of the executed assessment workflow and it is also
+   * the main content for the NOA (Notice of Assessment).
+   */
+  @Column({
+    name: "assessment_data",
+    type: "jsonb",
+    nullable: true,
```

Comment:
![image](https://user-images.githubusercontent.com/77353155/157138203-4eefcd27-5a07-4337-adc2-04a8044f0c4a.png) contradicting

Reply:
The table declaration is wrong, I will change it.